src/include/network/lan/wizards.rb

```diff
@@ -55,11 +55,12 @@ def LanSequence
       aliases = {
         "read"     => [-> { ReadDialog() }, true],
         "main"     => -> { MainSequence("") },
-        "packages" => [-> { PackagesInstall(Lan.Packages) }, true],
         "write"    => [-> { WriteDialog() }, true]
```

Comment:
This change makes RuboCop fail with "Align the elements of a hash literal if they span more than one line."
W!T!F!